docs/pages/kubernetes-access/helm/reference/teleport-plugin-mattermost.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: teleport-plugin-mattermost Chart Reference
+description: Values that can be set using the teleport-plugin-mattermost Helm chart
+---
+
+The `teleport-plugin-mattermost` Helm chart is used to configure the MatterMost Teleport plugin which allows users to receive access requests to channels or as direct messages via MatterMost.
+
+You can [browse the source on GitHub](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/mattermost).
+
+This reference details available values for the `teleport-plugin-mattermost` chart.
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## `teleport.address`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+This parameter contains the host/port combination of the teleport auth server.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretName`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Name of the Kubernetes secret that contains the credentials for the connection.
+
+The secret should be in the following format:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: teleport-plugin-mattermost-identity
+data:
+  auth_id: ...
```

Comment:
I wonder if it may be worth indicating that this field needs to be base64 encoded, or use the `stringData` field ? I know it's pretty obvious for people familiar with Kubernetes, but can easily be a stumbling point for less familiar operators who are following instructions precisely.

Reply:
If they followed the directions above it should automatically base64 encode the data. However, if they are creating the secret manually using the above secret they'll probably get a `BadRequest` error warning them it isn't base64 encoded.

It wouldn't hurt though to put <base64 encoded data...> or something like that there instead of ...